spec/spec_helper.rb

```diff
@@ -57,7 +57,7 @@ def puts(payload)
     # Some tests mess with the settings. This ensures one test cannot pollute another
     LogStash::SETTINGS.reset
 
-    LogStash::SETTINGS.set("queue.type", "memory_acked")
+    LogStash::SETTINGS.set("queue.type", "persisted")
```

Comment:
On `master`, `6.x`, and `6.3`, we're currently using `memory` in specs for testing stability reasons (see: https://github.com/elastic/logstash/pull/9601); do we want to do that here too?

Reply:
@yaauie this was on purpose, trying to get a little extra coverage given the circumstances :)